src/Pages/Form/Containers/FormContainers.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+import SectionTitle from "../Components/SectionTitle";
+
+const FormContainers: React.FC = () => {
+  return <SectionTitle title="Patient Information" />;
```

Comment:
Actually, this component will be implemented in Section component. So, this line can be removed and replaced with Fragment(<></>)